manifests/repo.pp

```diff
@@ -188,7 +188,7 @@
         logoutput => on_failure,
       }
 
-      if $::mrepo_env {
+      if $mrepo_env {
```

Comment:
Done.  I'm gonna avoid looking too deeply at the code just right now.  For instance, to use the parameter `mrepo_logging`, am I really expected to set it to a string like `> /path/to/logfile.txt`??

and `mrepo_env` is just a string that gets rammed in front of the command instead of being passed as the `environment` argument in the `Exec`??